src/Adapters/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from "./IAdapter";
+export * from "./MemoryAdapter";
 export * from "./MockAdapter";
```

Comment:
Based on the changes to all the imports below, did you mean to remove `MockAdapter` from being exported here?

Reply:
I had to update the import for `MockAdapter` to import from the file directly rather than the index due to a cyclical reference it was creating, causing issues with our webpack TypeScript loader not properly including the dependencies when it's transpiled. I'm wary of not exporting it from the index entirely as that would make it inaccessible to consumers of this library.